driver-async/src/main/com/mongodb/async/client/AbstractSubscription.java

```diff
@@ -125,7 +125,7 @@ void onError(final Throwable t) {
     }
 
     void onNext(final TResult next) {
-        boolean isTerminated = false;
+        boolean isTerminated;
```

Comment:
The initializations in this method were reported as extraneous by IntelliJ

Reply:
Ack